recipes-extended/xen/files/libxl-openxt-helpers.patch

```diff
@@ -113,51 +113,79 @@ PATCHES
      libxl__store_libxl_entry(gc, dm_domid, "stubdom-version",
 --- a/tools/libxl/libxl_domain.c
 +++ b/tools/libxl/libxl_domain.c
-@@ -1023,6 +1023,7 @@ void libxl__destroy_domid(libxl__egc *eg
+@@ -1020,11 +1020,15 @@ static void domain_destroy_domid_cb(libx
+                                     libxl__ev_child *destroyer,
+                                     pid_t pid, int status);
+ 
++/* OpenXT functions */
```

Comment:
Would be helpful to have a comment in the PR description, or a separate JIRA ticket, describing why these changes were necessary.  Are they specific to the 4.12 uprev, or fixing a general issue?

Reply:
I should have added comments to all of these, will keep that in mind for the future. All of these changes are specific to the 4.12 uprev (added to PR description). For this diff in particular, it was another (mostly) contextual adjustment. I grouped the openxt helper bits in their own function, kill_openxt_helpers, and moved the call inside of a function that was added in 4.12. The changes should functionally be the same as before (kill the openxt helpers if the domain being destroyed has a stubdom).